src/js/ConfigInserter.js

```diff
@@ -35,24 +35,23 @@ function readUint32(firmware, index) {
 }
 
 function getCustomDefaultsArea(firmware) {
-    var result = {};
+    const result = {};
 
-    var index = seek(firmware, CUSTOM_DEFAULTS_POINTER_ADDRESS);
+    const index = seek(firmware, CUSTOM_DEFAULTS_POINTER_ADDRESS);
 
-    if (index.byteIndex === undefined) {
-        return;
-    }
+    if (!index?.byteIndex) {
```

Comment:
There is no way that `seek()` can return `undefined`, so using optional chaining here is a bug, as it will hide if something goes wrong.

Reply:
Sonar incompliant. Had to rewrite it this way instead checking if undefined